spec/ics-027-direct-interchain-exchange/ReadMe.md

```diff
@@ -0,0 +1,74 @@
+# Cosmos Direct Interchain Exchange Protocol (DIEP)
+
+
+#### Core Idea:
+
+Asynchronous liquidity available with IBC that is resistant to theft by misbehaving validators. This eliminates the need to concentrate liquidity (limit orders) in a third party blockchain/venue. Price discovery and execution becomes native between Cosmos zones (e.g. N^2 IBC connections for N zones). There is a cost in terms of higher latency from this architecture.
```

Comment:
Okay so is this _also_ meant to support shared staking cross chain? Aka validator-A on chain-A can use funds locked for staking on Chain-A to act as staking tokens to also validate on Chain-B? If so we should more explicitly mention this in this initial paragraph

_________________________

edit: it seems as though this is not specifically intended, but I think there may be a way to have share some fundamental infrastructure of this `DIPER` protocol and the shared-collateralization.